tests/e2e/tools/lib/media-library.js

```diff
@@ -0,0 +1,59 @@
+import {expect} from './test-utils.js';
+
+/**
+ * Searches for images in the Library based on a search term or image id.
+ *
+ * @param {Object} page       - The page object for interacting with the browser.
+ * @param {string} searchTerm - The term to be searched, can be an image id.
+ */
+async function searchImages(page, searchTerm) {
+  const searchInput = page.getByPlaceholder('Search');
+
+  await searchInput.fill(searchTerm);
+  await page.keyboard.press('Enter');
+  await waitForLibraryLoad(page);
+}
+
+/**
+ * Deletes an image from the Library.
+ *
+ * @param {Object} page    - The page object for interacting with the browser.
+ * @param {string} imageId - The image to be deleted.
+ */
+async function deleteImageFromLibrary(page, imageId) {
+  const sidebar = page.locator('.archive-picker-sidebar');
+  if (imageId) {
+    // Search for the image.
+    await searchImages(page, imageId);
+    const image = page.locator('.picker-list > li');
+
+    // Select the image and open its link via the sidebar.
+    await image.click();
+    await expect(sidebar).toBeVisible();
+  }
```

Comment:
What about using a guard clause here?
If no `imageId` is provided, I think we should not execute anything else.

```suggestion
  if (!imageId) return;
  
  // Search for the image.
    await searchImages(page, imageId);
    const image = page.locator('.picker-list > li');

    // Select the image and open its link via the sidebar.
    await image.click();
    await expect(sidebar).toBeVisible();
```

Reply:
Actually with the way it is implemented and used now, we should still execute the following lines, because in the first test the image is already selected hence why we don't provide the image id and don't need to search for it... I hope that makes sense 😅

Reply:
It makes sense!
In that case, should we turn `imageId` into an optional parameter and explain in the JSDocs that in some cases passing it is not necessary?